api/openapi/specs/common/resource.yaml

```diff
@@ -174,9 +174,11 @@ components:
           type: string
         conf:
           description: The actual conf generated
-          type: object
-          additionalProperties: true
-          x-go-type: 'interface{}'
+          type: array
```

Comment:
This is the source of truth for this type, right? It's not generated from go types or something?

Reply:
Yeah, this the single source of truth. I found out this generates the go types by running `make generate/oas`, but then you need to generate the complete bundled OpenAPI (the one in `/docs`) which we use for the Typescript generation, hence the new make target.